saleor/graphql/product/mutations/products.py

```diff
@@ -1320,6 +1332,7 @@ def save(cls, info, instance, cleaned_input):
             AttributeAssignmentMixin.save(instance, attributes)
             instance.name = generate_name_for_variant(instance)
             instance.save(update_fields=["name"])
+        info.context.plugins.product_updated(instance.product)
```

Comment:
It seems that there are a lot of places where we could prefetch the related objects. Maybe we could move it `PluginManager.product_updated` ? What do you think?